### ISSUE TITLE
Help Pages do not open

### ISSUE THREAD

**Issue by patrickgonz** [IBMa/AAT#13]
Hello,

DAP help pages are not working with the new accessibility checklist that was release on February 2019. I am using a new token, but the page fails to load when I click on the ? icon on a potential violation. I get an error that that it cannot GET the page.

Thanks, Patrick

**Comment by asghar3** [comment]
Hi @patrickgonz 

I tired running DAP version 1.0.0.7 from https://aat.mybluemix.net.
I am able to scan without issues and also look at the help files by clicking on the help icon beside on each violation.

Are you still running into this issue ? We haven't touched the DAP recently.
You shouldn't be running into any issues.
Let us know. Thanks
